modules/rn_3_80.adoc

```diff
@@ -0,0 +1,148 @@
+
+//This will eventually expand to cover the latest three releases. Since this is the first TP tracker, it will include only 3.6.
+
+.Technology Preview tracker
+[cols="4,1,1,1",options="header"]
+|===
+|Feature | Quay 3.8 | Quay 3.7 |Quay 3.6
+
+| IPv6 Support
+| General Availability 
+| -
+| -
+
+| LDAP_SUPERUSER_FILTER
+| General Availability 
+|-
+|- 
+
+| FEATURE_SUPERUSERS_FULL_ACCESS
+| General Availability 
+|-
+|-
+
+| FEATURE_RESTRICTED_USERS
+| General Availability 
+|-
+|-
+
+| RESTRICTED_USERS_WHITELIST
+| General Availability 
+|-
+|-
+ 
+
+|link:https://access.redhat.com//documentation/en-us/red_hat_quay/3.7/html-single/use_red_hat_quay#red-hat-quay-quota-management-and-enforcement[Quota management and enforcement]
+|General Availability
+|General Availability
+|-
+
+
+|link:https://access.redhat.com/documentation/en-us/red_hat_quay/3.7/html-single/use_red_hat_quay#red-hat-quay-builders-enhancement[{productname} build enhancements]
+|General Availability
+|General Availability
+|-
+
+|link:https://access.redhat.com/documentation/en-us/red_hat_quay/3.7/html-single/use_red_hat_quay#quay-as-cache-proxy[{productname} as proxy cache for upstream registries]
+|General Availability 
+|Technology Preview
+|-
+
+|link:https://access.redhat.com/documentation/en-us/red_hat_quay/3.7/html-single/deploy_red_hat_quay_on_openshift_with_the_quay_operator/index[Geo-replication - {productname} Operator]
+|General Availability
+|General Availability
+|-
+
+
+|link:https://access.redhat.com/documentation/en-us/red_hat_quay/3.7/html-single/manage_red_hat_quay#unmanaged_clair_configuration[Advanced Clair configuration]
+|General Availability
+|General Availability
+|-
+
+|Support for Microsoft Azure Government (MAG)
+|General Availability
+|General Availability
+|-
+
+|link:https://access.redhat.com/documentation/en-us/red_hat_quay/{producty}/html-single/configure_red_hat_quay/index#config-fields-helm-oci[FEATURE_HELM_OCI_SUPPORT]
+|Deprecated
+|Deprecated
+|Deprecated
+
+|link:https://access.redhat.com/documentation/en-us/red_hat_quay/{producty}/html-single/configure_red_hat_quay/index#config-ui-database[MySQL and MariaDB database support]
+|Deprecated
+|Deprecated
+|Deprecated
+
+|link:https://access.redhat.com/documentation/en-us/red_hat_quay/{producty}/html-single/use_red_hat_quay/index#oci-intro[Open Container Initiative (OCI) Media types]
+|General Availability
+|General Availability
+|General Availability
+
+|link:https://access.redhat.com/documentation/en-us/red_hat_quay/3.7/html-single/manage_red_hat_quay#clair_crda_configuration[Java scanning with Clair]
+|Technology Preview
+|Technology Preview
+|Technology Preview
```

Comment:
Just an observation between quay and ocp tech preview table: I like that this one is written out as Technology Preview etc instead of TP. Looks cleaner imo!